confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -123,26 +132,94 @@ public TSStatus setStorageGroup(PhysicalPlan physicalPlan) {
   }
 
   @Override
-  public DataSet getSchemaPartition(PhysicalPlan physicalPlan) {
+  public DataSet getSchemaPartition(PathPatternTree patternTree) {
+    TSStatus status = confirmLeader();
```

Comment:
getSchemaPartition should connect all node ,include follwer node.   follwer node return result.

Reply:
For now, we first give all read and write requests to the leader for execution, and then further optimize with the consensus layer.